examples/offscreen-canvas.js

```diff
@@ -120,6 +131,8 @@ worker.addEventListener('message', (message) => {
       );
     });
     image.src = event.data.src;
+  } else if (message.data.action === 'getFeatures') {
+    console.log(message.data.features); // eslint-disable-line
```

Comment:
Would be nice if you could display the hit information in the actual web page in / below the map instead of logging it.
Maybe just the information if a feature is at the pointer position, or the type of feature to keep it simple.

Reply:
I now added info window from example vector tile info